base/shell/explorer/settings.cpp

```diff
@@ -61,6 +62,9 @@ BOOL TaskbarSettings::Load()
     dwRet = SHGetValueW(hkExplorer, L"Advanced", L"TaskbarSmallIcons", NULL, &dwValue, &cbSize);
     bSmallIcons = (dwRet == ERROR_SUCCESS) ? (dwValue != 0) : TRUE;
 
+    dwRet = SHGetValueW(hkExplorer, L"Advanced", L"TaskbarSd", NULL, &dwValue, &cbSize);
```

Comment:
Just to know, this is a ROS-only setting?

Reply:
This is a ROS-only setting, but it is the same registry value that we had been using before to turn on or off the show desktop button. Personally, I would like to rename it to `ShowDesktopButton`. For compatibility though, I kept it as `TaskbarSd`.